test/functional/content_view/update_test.rb

```diff
@@ -48,6 +48,30 @@ module HammerCLIKatello
         end
         run_cmd(%w(content-view update --name cv2 --organization-label org1))
       end
+
+      it 'requires organization if product name is supplied along with repository' do
+        id = 122
+        repo_id = 100
+        product_name = "foo"
+        api_expects_no_call
+        cmd = %W(content-view update --id=#{id}\
+                 --repository-ids=#{repo_id} --product=#{product_name})
+        result = run_cmd(cmd)
+        assert(result.err[/--organization-id, --organization, --organization-label is required/],
+               "Organization option requirements are validated")
+      end
+
+      it 'requires product if repository names are provided' do
+        id = 122
+        repo_id = 100
+        organization_id = 5
+        api_expects_no_call
+        cmd = %W(content-view update --id=#{id}\
+                 --repositories=#{repo_id} --organization-id=#{organization_id})
+        result = run_cmd(cmd)
+        assert(result.err[/--product-id, --product is required/],
+               "Product option requirements are validated")
```

Comment:
This string is shown in the test output when this test fails. IMO it should describe what went wrong, [like this](https://github.com/Katello/hammer-cli-katello/blob/49b48b3c1d431da10c254e3c59e62552cb909122/test/functional/package/list_test.rb#L34).